app/__init__.py

```diff
@@ -17,8 +18,9 @@ def create_app():
         app.config.from_object("config.Config")
 
     db.init_app(app)
-    CKEditor(app)
+    ckeditor.init_app(app)
```

Comment:
Does this change imply you got CKEditor working?

Reply:
Yes! 😅 it is working but it displays all of the fields.